docs/topics/mta-7-installing-web-console-on-ocp-local.adoc

```diff
@@ -0,0 +1,285 @@
+
+[id="mta-7-ocp-local-windows-reqs_{context}"]
+=== {rh-ocp-local} requirements on Microsoft Windows
+
+On Microsoft Windows, {rh-ocp-local} requires the Windows 10 Fall Creators Update (version 1709) or later. {rh-ocp-local} does not run on earlier versions of Microsoft Windows. Microsoft Windows 10 Home Edition is not supported.
+
+[id="mta-7-ocp-local-macos-reqs_{context}"]
+=== {rh-ocp-local} requirements on macOS
+
+On macOS, {rh-ocp-local} requires macOS 11 Big Sur or later. {rh-ocp-local} does not run on earlier versions of macOS.
+
+[id="mta-7-ocp-local-local-reqs_{context}"]
+=== {rh-ocp-local} requirements on Linux
+
+On Linux, {rh-ocp-local} is supported only on the latest two Red Hat Enterprise Linux 8 and 9 minor releases and on the latest two stable Fedora releases.
+
+When using Red Hat Enterprise Linux, the machine running {rh-ocp-local} must be registered with the Red Hat Customer Portal.
+
+Ubuntu 18.04 LTS or later and Debian 10 or later are not supported and might require manual setup of the host machine.
+
+==== Required software packages for Linux
+
+{rh-ocp-local} requires the `libvirt` and `NetworkManager` packages to run on Linux:
+
+* On *Fedora* and *Red Hat Enterprise Linux* run:
+
++
+[source,terminal]
+----
+sudo dnf install NetworkManager
+----
+
+* *Debian/Ubuntu* run:
++
+[source,terminal]
+----
+sudo apt install qemu-kvm libvirt-daemon libvirt-daemon-system network-manager
+----
+
+[id="mta-7-installing-ocp-local_{context}"]
+== Installing the {ProductName} Operator in a {rh-ocp-local} environment
+
+To install OpenShift Local:
+
+. Download the latest release of link:https://console.redhat.com/openshift/create/local[{rh-ocp-local}]( for your platform.
+
+.. Download *OpenShift Local*.
+
+.. Download *pull secret*.
+
+. Assuming you saved the archive in the `~/Downloads directory` follow these steps:
+
++
+[source,terminal]
+----
+cd ~/Downloads
+----
+
++
+[source,terminal]
+----
+tar xvf crc-linux-amd64.tar.xz
+----
+
+. Copy the `crc` executable to it:
+
++
+[source,terminal]
+----
+cp ~/Downloads/crc-linux-<version-number>-amd64/crc ~/bin/crc
+----
+. Add the `~/bin/crc` directory to your `$PATH` variable:
+
++
+[source,terminal]
+----
+export PATH=$PATH:$HOME/bin/crc
+----
++
+[source,terminal]
+----
+echo 'export PATH=$PATH:$HOME/bin/crc' >> ~/.bashrc
+----
+
+. To disable telemetry, run the following command:
+
++
+[source,terminal]
+----
+crc config set consent-telemetry no
+----
+
+[NOTE]
+====
+For macOS, download the relevant *crc-macos-installer.pkg*.
+
+. Navigate to *Downloads* using *Finder*.
+. Double-click on `crc-macos-installer.pkg`.
+====
+
+[id="mta-7-set-up-ocp-local_{context}"]
+== Setting up {rh-ocp-local}
+
+The `crc setup` command performs operations to set up the environment of your host machine for the {rh-ocp-local} instance.
+
+The `crc setup` command creates the `~/.crc directory`
+
+1. Set up your host machine for OpenShift Local:
+
```

Comment:
Line 122:  Suggestions: Remove "1." and use the attribute for "OpenShift local".